cmd/query/app/apiv3/otlp_translator.go

```diff
@@ -45,3 +47,26 @@ func modelToOTLP(spans []*model.Span) ([]*tracev1.ResourceSpans, error) {
 	}
 	return chunk.ResourceSpans, nil
 }
+
+func OTLP2model(OTLPSpans []byte) ([]model.Trace, error) {
+	ptrace_unmarshaler := ptrace.JSONUnmarshaler{}
+	otlp_traces, err := ptrace_unmarshaler.UnmarshalTraces(OTLPSpans)
```

Comment:
```suggestion
	ptraceUnmarshaler := ptrace.JSONUnmarshaler{}
	otlpTraces, err := ptrace_unmarshaler.UnmarshalTraces(OTLPSpans)
```
same applies to `jaeger_traces` and `jaeger_traces`

context: https://go.dev/doc/effective_go#mixed-caps